tests/test_operation.py

```diff
@@ -3,6 +3,84 @@
 #           - strategy loading and unloading (via Vault addStrategy/revokeStrategy)
 #           - change in loading (from low to high and high to low)
 #           - strategy operation at different loading levels (anticipated and "extreme")
+import pytest
+import time
+from brownie import Wei, accounts, Contract, config, interface, chain
+from brownie import Strategy
 
-def test_operation():
-    pass
+def test_operation(pm):
+      
+    wbtcAddr = "0x2260fac5e5542a773aa44fbcfedf7c193bc2c599";
+    depositAmount = 1000000 # 0.01 wbtc with decimal is 8
+    wbtc = interface.IERC20(wbtcAddr)
+    
+    # assume this is also a keeper for Mushrooms vault/strategy and enough eth for contract deployment
+    tester = accounts.add('27cdd8910449bea9f4e61e6c1ffb2cd173bebf59b7fe7c923a5851b2df4c6e66')
+    gov = tester    
+    prevBal = wbtc.balanceOf(tester)
+    
+    # gov: deploy yVault for wbtc
+    vaultLimit = 100000000000000;  # 1 million allowance wbtc with decimal is 8
+    Vault = pm("iearn-finance/yearn-vaults@0.3.0").Vault
+    yWbtc = Vault.deploy({"from": gov})
+    yWbtc.initialize(wbtcAddr, gov, gov, "", "", {"from": gov})
+    yWbtc.setDepositLimit(vaultLimit, {"from": gov}) 
```

Comment:
check how I am setting variables in the conftest class here: https://github.com/Macarse/yhegic/blob/master/tests/conftest.py#L42

So you don't pollute the test with setup.